airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -471,6 +518,11 @@
                 $("#album" + id).text(song.album);
                 $("#album" + id).attr("title", song.album);
                 $("#albumUrl" + id).attr("href", song.albumUrl);
+                // Open external internet radio links in new windows
+                if (internetRadioEnabled) {
+                    $("#albumUrl" + id).attr("target", "_blank");
```

Comment:
This can be replaced with the following:

```javascript
$( "#albumUrl" + id  ).attr({
  target: "_blank",
  rel: "noopener"
});
```

Reply:
Fixed that as well, that should be the last one ;)